<launch>
    <node pkg="potdetect" type="potdetect" name="potdetect" output="screen" />
    <node pkg="land" type="landTargetPosition" name="landTargetPosition" output="screen" />
    
    <node pkg="imav" type="gameplan_barrel" name="gameplan_barrel" output="screen">
        <param name="height_land" type="double" value="1.2" />
        <param name="counter_to_savebarrel" type="int" value="10" />
        <param name="counter_to_checkbarrel" type="int" value="5" />
    </node>
</launch>
